src/lib/publisher/term/change/index.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  PublisherTermChangeCancelParams,
+  PublisherTermChangeGetSubscriptionUpgradeStatusParams
+} from '../../../interfaces/api-params'
+import {
+  PublisherTermChangeCancelResponse,
+  PublisherTermChangeGetSubscriptionUpgradeStatusResponse
+} from '../../../interfaces/api-response'
+import { SubscriptionUpgradeStatus as ISubscriptionUpgradeStatus } from '../../../interfaces/subscription-upgrade-status';
+import { Piano } from '../../../piano';
+import { httpRequest } from '../../../utils/http-request';
+
+const ENDPOINT_PATH_PREFIX = '/publisher/term/change';
+
+export class Change {
+  constructor (private readonly piano: Piano) {}
+
+  /**
+   * Get upgrade status of subscription
+   *
+   * @see https://docs.piano.io/api/?endpoint=post~2F~2Fpublisher~2Fterm~2Fchange~2FgetSubscriptionUpgradeStatus
+   */
+
```

Comment:
nits: this empty line should be removed.

Reply:
@5t111111 Thanks for the catch! Fixed here: [a5ce9db](https://github.com/kodansha/piano-sdk-for-nodejs/pull/14/commits/a5ce9dba7e9c2443a575e58ae3113a87663df25c)